scripts/ft-benchmark-data-sender.py

```diff
@@ -0,0 +1,152 @@
+    if response.status_code != 200:
+        raise f"Failed to retrieve commit hash. HTTP Status code: {response.status_code}"
+    version = response.json()["result"]["version"]["build"]
+    short_commit = version.split("-")[2][1:]
+    github_url = f"https://api.github.com/repos/near/nearcore/commits/{short_commit}"
+    response = requests.get(github_url)
+    commit_data = response.json()
+    full_commit_hash = commit_data["sha"]
+    commit_timestamp_str = commit_data["commit"]["author"]["date"]
+    commit_timestamp = datetime.strptime(commit_timestamp_str,
+                                         "%Y-%m-%dT%H:%M:%SZ")
+    return (full_commit_hash, commit_timestamp)
+
+
+# TODO: Make accesses actually work
+def commit_to_db(data: dict) -> None:
+    with psycopg2.connect(
+            dbname="benchmarks",
+            user="node-data-sender",
+            password=getenv("DB_PASSWORD"),
+            host="34.90.190.128",
+            port="5432",
+    ) as conn:
+        with conn.cursor() as cursor:
+            insert_query = sql.SQL("""
+                INSERT INTO ft_transfer (
+                    time, git_commit_hash, git_commit_time, num_nodes, node_hardware, 
+                    num_traffic_gen_machines, disjoint_workloads, num_shards, 
+                    num_unique_users, size_state_bytes, tps, total_transactions
+                ) VALUES (
+                    %(time)s, %(git_commit_hash)s, %(git_commit_time)s, %(num_nodes)s, 
+                    %(node_hardware)s, %(num_traffic_gen_machines)s, %(disjoint_workloads)s, 
+                    %(num_shards)s, %(num_unique_users)s, %(size_state_bytes)s, 
+                    %(tps)s, %(total_transactions)s
+                )
+                """)
+            cursor.execute(insert_query, data)
+            conn.commit()
+
+
+# TODO: send signal to this process if ft-benchmark.sh decided to switch neard to another commit.
+# add handling of this signal to this script
+if __name__ == "__main__":
+    state_size = (int(
+        subprocess.check_output(["du", "-s", "~/.near/localnet/node0/data"
+                                ]).decode("utf-8").split()[0]) * 1024)
+    processed_transactions = []
+    time_begin = datetime.now()
+    while True:
+        if (datetime.now() - time_begin).seconds / 3600 > DURATION:
+            break
+        processed_transactions.append(calculate_processed_transactions())
+        print("Added transaction count to list")
+        sleep(POLL_INTERVAL)
+    processed_transactions_deltas = np.diff(processed_transactions)
+    processed_transactions_deltas = np.array(
+        list(map(lambda x: x / POLL_INTERVAL, processed_transactions_deltas)))
+    average_tps = np.mean(processed_transactions_deltas)
+    variance_tps = np.var(processed_transactions_deltas)
+    # TODO: will be good to have all "probably should be filled by terraform" as command line arguments
+    responce = {
+        "time": time_begin,
```

Comment:
Let's store both `start_time` and `end_time`

Reply:
Then we also need to change `ft_transfer_db_schema.sql`

Reply:
Yes, I think this should be reasonably simple to do. CC @mooori to help out with this migration.

Doesn't have to happen for this PR to land though, let's just add a TODO.